media-gfx/flameshot/files/flameshot-0.10.1-dont-force-ccache.patch

```diff
@@ -0,0 +1,13 @@
+diff -ruN a/CMakeLists.txt b/CMakeLists.txt
+--- a/CMakeLists.txt	2021-05-03 12:55:07.561259897 +0300
++++ b/CMakeLists.txt	2021-05-03 12:55:36.772461830 +0300
+@@ -70,9 +70,6 @@
+ 
+ add_library(project_warnings INTERFACE)
+ 
+-# enable cache system
+-include(cmake/Cache.cmake)
+-
+ # standard compiler warnings
+ include(cmake/CompilerWarnings.cmake)
+ # set_project_warnings(project_warnings)
```

Comment:
I'd mention [cleaning useless date/diff cruft](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto), but I don't think this patch is needed.

Looking at Cache.cmake, it let you pass `-DENABLE_CACHE=0` to disable instead.